docs/charmed_k8s.md

```diff
@@ -0,0 +1,47 @@
+## Working with Charmed Kubernetes
+
+### Setup
+
+Assuming you have juju and charmed kubernetes already setup, let's discuss the steps to launch Spark jobs against the charmed kubernetes setup.
+
+First thing is to set up the spark-client snap
+```shell
+$ sudo snap install spark-client --edge
+```
+
+The snap needs to point to the charmed kubernetes setup, so let's set that up as well.
+
+```shell
+$ mkdir ~/.kube
+$ juju scp kubernetes-control-plane/0:config ~/.kube/config
```

Comment:
Ugh. Isn't there any better way to do this?

Reply:
After inputs from Rob and Mykola, we decided to point to the official documentation of setting up charmed kubernetes. That way we always have the up to date instructions for doing this.